app/src/pages/inside/profilePage/localizationBlock/localizationBlock.jsx

```diff
@@ -78,6 +78,16 @@ const messages = defineMessages({
   },
 });
 
+const langName = {
```

Comment:
Maybe it is better to move that object to Profile page events?

Reply:
I think it's more logical to leave the object with languages in the localizationBlock.jsx file than in ProfilePageEvents(analytic file).
And I didn't find the constants, could you show me where they are?
We have constants with codes, but not with language names (names in English)

Reply:
That is what I mean.

Reply:
you correct! fixed ✅